---a/net/minecraft/scoreboard/ScoreObjective.java
+++b/net/minecraft/scoreboard/ScoreObjective.java
@@ -1,10 +1,6 @@
 package net.minecraft.scoreboard;
 
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class ScoreObjective
-{
+public class ScoreObjective {
+
     private final Scoreboard theScoreboard;
     private final String name;
     private final IScoreObjectiveCriteria objectiveCriteria;
@@ -12,8 +12,7 @@
     private String displayName;
     private static final String __OBFID = "CL_00000614";
 
-    public ScoreObjective(Scoreboard p_i2307_1_, String p_i2307_2_, IScoreObjectiveCriteria p_i2307_3_)
-    {
+    public ScoreObjective(Scoreboard p_i2307_1_, String p_i2307_2_, IScoreObjectiveCriteria p_i2307_3_) {
         this.theScoreboard = p_i2307_1_;
         this.name = p_i2307_2_;
         this.objectiveCriteria = p_i2307_3_;
@@ -21,48 +21,29 @@
         this.renderType = p_i2307_3_.getRenderType();
     }
 
-    @SideOnly(Side.CLIENT)
-    public Scoreboard getScoreboard()
-    {
-        return this.theScoreboard;
-    }
-
-    public String getName()
-    {
+    public String getName() {
         return this.name;
     }
 
-    public IScoreObjectiveCriteria getCriteria()
-    {
+    public IScoreObjectiveCriteria getCriteria() {
         return this.objectiveCriteria;
     }
 
-    public String getDisplayName()
-    {
+    public String getDisplayName() {
         return this.displayName;
     }
 
-    public void setDisplayName(String p_96681_1_)
-    {
+    public void setDisplayName(String p_96681_1_) {
         this.displayName = p_96681_1_;
         this.theScoreboard.func_96532_b(this);
     }
 
-    public IScoreObjectiveCriteria.EnumRenderType getRenderType()
-    {
+    public IScoreObjectiveCriteria.EnumRenderType getRenderType() {
         return this.renderType;
     }
 
-    public void setRenderType(IScoreObjectiveCriteria.EnumRenderType type)
-    {
+    public void setRenderType(IScoreObjectiveCriteria.EnumRenderType type) {
         this.renderType = type;
         this.theScoreboard.func_96532_b(this);
     }
-}
+}
